### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by MarkVed17**
~ Added the landing page
~ Added the required assets
~ Added the common CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+<body>
+  <div>
+    <!-- Navbar -->
+    <header class="header-nav nav-search">
+      <a href="index.html" class="header-logo-link">
+        <img class="header-logo-img" src="/assets/movies-logo/Home_Camp_Cretaceous_Logo.svg" alt="jurassic-park-logo" />
+      </a>
+
+      <div class="std-search">
+        <input type="text" placeholder="Search..." class="input standard nav-searchbar" />
```
Comment (sanathkumar26700) [inline index.html:27]: suggestion :  can use type as "search"

**Review by sanathkumar26700 — APPROVED** [review]
✅ Clean code
✅ Unique UI
✅ Everything works fine

**Review by percius47 — APPROVED** [review]

File: index.css [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+  letter-spacing: 5px;
+  word-spacing: 0.5rem;
+  font-size: 4.5rem;
+  color: var(--font-color);
+  flex-basis: 100%;
+}
+
+.home-screen {
+  display: flex;
+  flex-direction: column;
```
Comment (msdeshmukh009) [inline index.css:32]: I see you are repeating these styles in multiple places consider making utility class for it.
```html
  display: flex;
  flex-direction: column;
```

**Review by msdeshmukh009 — APPROVED** [review]
The site is looking 🔥. Suggestion: try to find Webp format images as Webp images are on average 27% lighter than PNG.

**Pull request merged**